src/devices/Arduino/FirmataDevice.cs

```diff
@@ -0,0 +1,1234 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using UnitsNet;
+
+namespace Iot.Device.Arduino
+{
+    /// <summary>
+    /// This delegate is used to fire digital value changed events.
+    /// </summary>
+    public delegate void DigitalPinValueChanged(int pin, PinValue newValue);
```

Comment:
Could this use https://github.com/dotnet/iot/blob/b5bb684bbf00ebd266018cfffde3dcbde26f0538/src/System.Device.Gpio/System/Device/Gpio/PinChangeEventHandler.cs instead?

Reply:
Done. This is internal only, anyway.